src/App.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import './App.css';
+import { Box, Typography } from '@mui/joy';
+import ResponsiveAppBar from './components/ResponsiveAppBar';
+import CompoundForm from './components/form/CompoundForm';
+import { formatSums } from './utils/helpers';
+import SumsCard from './components/ui/SumsCard';
+//-------------------------------------------------
+// interface ParentProps {
+//   sendDataToParent: (data: {
+//     futureValue: number;
+//     totalInterest: number;
+//     futureValueArray: number[];
+//   }) => void;
+// }
+//-------------------------------------------------
+
+function App() {
+  const [futureValue, setFutureValue] = useState<number | string>('');
```

Comment:
number | string? futureValue is a `number` in your code 
avoid multiple types if not a must